.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -0,0 +1,43 @@
+# .NET Core Release Checklist
+
+## Release Versions
+
+_The set of .NET Core versions that are being released as a unit._
+
+* 2.1.X: <runtime/SDK>
+* 3.1.X: <runtime/SDK>
+
+
+
+1. - [ ] Incase of 2.x, run auto-update(refer docs). Incase of 3.X:
```

Comment:
> refer docs

Missing a link?

Reply:
I think I meant refer OneNote. Changing it.